websites/P/Phind/presence.ts

```diff
@@ -0,0 +1,94 @@
+const presence = new Presence({
+		clientId: "1106990410838065172",
+	}),
+	slideshow = presence.createSlideshow(),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const assets = {
+			phindLogo: "https://i.imgur.com/fakWcYA.png",
+			search: "https://i.imgur.com/a5qsEbL.png",
+		},
+		presenceData: PresenceData = {
+			largeImageKey: assets.phindLogo,
+			details: "Browsing Phind",
+		},
+		pathDetailsMap = {
+			"/default": "Viewing how to set Phind as default",
+			"/bangs": "Viewing !Bangs",
+			"/mobile": "Viewing Mobile Page",
+			"/hotkeys": "Viewing Hotkeys",
+			"/history": "Viewing Search History",
+			"/about": "Viewing About Page",
+			"/tutorial": "Viewing Tutorial",
+			"/privacy": "Viewing Privacy Policy",
+			"/terms": "Viewing Terms",
+		},
+		{ pathname, href } = document.location,
+		pathDetails = Object.entries(pathDetailsMap).find(([pathPrefix]) =>
+			pathname.startsWith(pathPrefix)
+		),
+		searchResults = document.querySelectorAll("div.row[name^='answer-']"),
+		[displayTime, displaySearch, cycleSearch, shareSearch, privateMode] =
+			await Promise.all([
+				presence.getSetting("displayTime"), // shows/hides the time spent on the site
+				presence.getSetting("displaySearch"), // shows/hides your search query text
+				presence.getSetting("cycleSearch"), // cycles through all the search query on the page
+				presence.getSetting("shareSearch"), // shares your search with a button
+				presence.getSetting("privateMode"), // hides the querry, button and what exact page you are on
```

Comment:
The comments here may be unneeded

Reply:
I think that if someone else reads these settings in the future, they may not be descriptive enough without the comments.
I'm a bit iffy about the naming anyways as I tried to make them fit into the Extension popout. In the code however, some different names might help a bit but the comments make sure they're 100% understood. Like the `cycle` and `share` consts.

But you can make the call, keep or delete?

Reply:
I think it's self explanatory enough.

>